tfx/components/example_gen/driver_test.py

```diff
@@ -57,61 +57,82 @@ def testResolveExecProperties(self):
             json_format.MessageToJson(
                 example_gen_pb2.Input(splits=[
                     example_gen_pb2.Input.Split(
-                        name='s1', pattern='span{SPAN}/split1/*'),
+                        name='s1',
+                        pattern='span{SPAN}/version{VERSION}/split1/*'),
```

Comment:
let's write the behaviors test for

1. have {span}, not {version}
2. have both {span} and {version}
3. have {version}, but not {span} 
4. have none of them

Reply:
you left this comment on the driver_tests, is that where these test scenarios should go, or would they go in the util_test file?

Reply:
as of now, I put these tests under `utils_test.py`, since the behavior we're testing is specifically in `utils.py`